runtime/tests/utils/utils.go

```diff
@@ -94,3 +94,18 @@ func DeploymentTransaction(name string, contract []byte) []byte {
 		hex.EncodeToString(contract),
 	))
 }
+
+func UpdateTransaction(name string, contract []byte) []byte {
```

Comment:
```suggestion
func ContractUpdateTransaction(name string, contract []byte) []byte {
```

Just a small suggestion, feel free to ignore :)